esp/templates/program/modules/availabilitymodule/availability_form.html

```diff
@@ -2,107 +2,50 @@
 
 {% block title %}Availability for {{ prog.niceName }}{% endblock %}
 
-{% block content %}
-
-{% load render_qsd %}
-
+{% block stylesheets %}
+{{ block.super }}
 <link rel="stylesheet" href="/media/styles/forms.css" type="text/css" />
+<link rel="stylesheet" href="/media/default_styles/availability.css" type="text/css" />
+{% endblock %}
 
-<style>
-.wrap {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-}
-
-.side {
-  width: 280px;
-}
-
-.summary {
-  display: block;
-  padding: 5px 5px;
-  font-size: 16px;
-  width: 100px;
-  margin: 5px 5px 5px 5px;
-  font-family: sans-serif;
-}
-
-.noselect {
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-}
-
-input,
-select {
-  padding: 2px;
-}
-
-td.group, table.group {
-  vertical-align: top !important;
-  border-spacing: 0px 2px !important;
-  padding: 0 !important;
-}
-
-#program_form table {
-  border-width: 0px;
-  border-collapse: separate;
-  border-spacing: 5px 1px;
-}
-
-#program_form th {
-  text-align: center;
-  font-weight: bold;
-  background: #EFEFEF;
-  padding: 3px;
-  height: 80px;
-  width: 75px;
-}
-
-.label {
-  text-align: right;
-  /* font-weight:bold; */
-  white-space: nowrap;
-  vertical-align: top;
-}
-
-.proposed, .canDo, .teaching {
-  font-size: 16px !important;
-  text-align: center;
-}
-
-.proposed {
-  background: #EFEFEF;
-}
+{% block xtrajs %}
+<script type="text/javascript" src="/media/scripts/program/modules/availability.js"></script>
 
-.canDo {
-  background: #00FF00;
-}
+<script>
+$j(document).ready(function() {
```

Comment:
Most of the stuff in this ready is actually specific to the teacher availability form, and there are analogous and very different functions for the volunteer form. This is because of the pretty different setup of the two original forms (for the teacher form we want to extract the data from the checkboxes and for the volunteer form we want to extract the labels for the checkboxes). I suppose I could move both versions to the JS file and specify which version to use in the `init` call? Let me see if there's a way to make the two forms more similar...